lib/model_configuration/model_configuration.rb

```diff
@@ -63,46 +53,33 @@ def skip_ui?
   end
 
   def show_index?
-    @show_index && !skip_ui?
+    !@skip_index && !skip_ui?
```

Comment:
I did consider this but I figured this was more consistent with how I'd describe the condition with speech